File: environment/pset6/cash/cash.py
from cs50 import get_float
while True:
    n=get_float("Change owed:")
    round(n)
    n*=100
    if(n>0):
        x=n%25
        minus = n-x
        z= minus/25

        u=x%10
        minus = x-u
        p= minus/10

        e=u%5
        minus = u-e
        o= minus/5

        g=e%1
        minus = e-g
        d= minus/1

        sum= z+p+o+d
        print(sum)

        break

File: environment/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
int x;
float n;
x=0;
double round(double n);
do
{
n=get_float();
}

while (n<0);
{
n *=100;

}
while (n>=25)
{x++;
 n -=25;
}
while (n>=10)
{x++;
n -=10;
}
while (n>=5)
{x++;
n -=5;
}
while (n>=1)
{x++;
n -=1;
}
printf("%i \n",x);
}